dasharo-compatibility/wifi-bluetooth-support.robot

```diff
@@ -30,61 +34,83 @@ WLE001.001 Wireless card detection (Ubuntu 20.04)
     Should Match    ${out}    *${WIFI_CARD_UBUNTU}*
+    Log To Console    The test passed for the ${WIFI_CARD_UBUNTU} wireless card
+    Log    The test passed for the ${WIFI_CARD_UBUNTU} wireless card    WARN
     Exit From Root User
 
 WLE001.002 Wireless card detection (Windows 11)
     [Documentation]    Check whether the Wi-Fi/Bluetooth card is enumerated
     ...    correctly and can be detected from the operating system.
     Skip If    not ${WIRELESS_CARD_SUPPORT}    WLE001.002 not supported
     Skip If    not ${TESTS_IN_WINDOWS_SUPPORT}    WLE001.002 not supported
+    Log To Console    Remember to test all variants of wireless cards.
+    Log    Remember to test all variants of wireless cards.    WARN
     Power On
     Login To Windows
     ${out}=    Execute Command In Terminal    Get-PnpDevice -PresentOnly | Select-String -Pattern "Wi-Fi"
-    Should Contain    ${out}    ${WIFI_CARD}
+    Should Match    ${out}    *${WIFI_CARD_UBUNTU}*
+    Log To Console    The test passed for the ${WIFI_CARD} wireless card
+    Log    The test passed for the ${WIFI_CARD} wireless card    WARN
 
 WLE002.001 Wi-Fi scanning (Ubuntu 20.04)
     [Documentation]    Check whether the Wi-Fi functionality of card is
     ...    initialized correctly and can be used from within the
     ...    operating system..
     Skip If    not ${WIRELESS_CARD_WIFI_SUPPORT}    WLE002.001 not supported
     Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    WLE002.001 not supported
+    Log To Console    Remember to test all variants of wireless cards.
+    Log    Remember to test all variants of wireless cards.    WARN
     Power On
     Boot System Or From Connected Disk    ubuntu
     Login To Linux
     Switch To Root User
     Scan For Wi-Fi In Linux
+    ${current_card}=    Execute Command In Terminal    lspci | grep "Network controller: | awk -F": " '{print $2}"
     Exit From Root User
+    Log To Console    The test passed for the ${current_card} wireless card
+    Log    The test passed for the ${current_card} wireless card    WARN
 
 WLE002.002 Wi-Fi scanning (Windows 11)
     [Documentation]    Check whether the Wi-Fi/Bluetooth card is enumerated
     ...    correctly and can be detected from the operating system.
     Skip If    not ${WIRELESS_CARD_WIFI_SUPPORT}    WLE002.002 not supported
     Skip If    not ${TESTS_IN_WINDOWS_SUPPORT}    WLE002.002 not supported
+    Log To Console    Remember to test all variants of wireless cards.
+    Log    Remember to test all variants of wireless cards.    WARN
     Power On
     Login To Windows
     Execute Command In Terminal    Start-Service WlanSvc
     ${out}=    Execute Command In Terminal    netsh wlan show network
     Should Contain    ${out}    3mdeb_abr
     Should Contain    ${out}    3mdeb_abr_5GHz
+    Log To Console    The test passed for the ${WIFI_CARD} wireless card
+    Log    The test passed for the ${WIFI_CARD} wireless card    WARN
```

Comment:
I don't know if it works, but apparently you can check the name of the wifi card.
https://learn.microsoft.com/en-us/powershell/module/netadapter/get-netadapter?view=windowsserver2022-ps

Reply:
[Implemented](https://github.com/Dasharo/open-source-firmware-validation/pull/136/files#diff-db99b3a0adebbc880d28cd1f87d49ea87c9b11cd1d3681f599bd1494c26e814cR61) for Windows test cases